src/library/scala/collection/concurrent/CNodeBase.java

```diff
@@ -19,15 +19,15 @@
     public volatile int csize = -1;
```

Comment:
Since this changes a file in a completely different package, you should put this, and other `TrieMap`-related changes into a separate commit.